docs/graphql/manual/migrations/advanced/seed-data-migration.rst

```diff
@@ -0,0 +1,61 @@
+Creating a seed data migration
+==============================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+It can be convenient to add data into tables as part of the DB init process. This section gives instructions as to how to achieve that.
+
+
+Step 1: Navigate to the SQL section
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the Hasura console, click on the ``Data`` tab and then on the ``SQL`` link on the left hand side.
+
+
+Step 2: Write an insert statement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The next step is to write an insert statement that populates the database with seed data, like this:
+
+.. code-block:: SQL
+
+    INSERT INTO addresses (id, location) VALUES
+    (1, 'Bangalore'),
+    (2, 'Tel Aviv'),
+    (3, 'Zurich');
+    INSERT INTO authors (id, name, address_id) VALUES
+    (1, 'Sarah', 3),
+    (2, 'Joey', 1),
+    (3, 'Rachel', 2);
+    INSERT INTO articles (id, title, content, author_id) VALUES
+    (1, 'How to make fajitas', 'Recipe on making the best fajitas in the world', 1),
+    (2, 'How to climb mount everest', 'Guide on successfully climbing the hightest mountain in the world', 3),
+    (3, 'How to be successful on brodway', 'What it takes for you to be a successful performer at broadway', 2);
+
+Step 3: Track the statement & create a migration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Check the box labelled ``Track this``, so that the data actually gets inserted into the DB. 
+
+- Check the box ``This is a migration`` and give it a migration name, e.g. ``insert_seed_data``.
+
+Step 4: Run the statement
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Hit the ``Run!`` button. 
+
+Step 4: Verify data & migration
```

Comment:
Step 5. I guess all the step numbers will change after we add the hasura console via cli step